packages/server/email/MailManagerMailgun.ts

```diff
@@ -30,4 +37,15 @@ export default class MailManagerMailgun extends MailManager {
     }
     return true
   }
+
+  async validateEmail(email: string) {
+    try {
+      const res = await this.mailgunClient.validate.get(email)
+      return res.result as MailgunValidateResult
+    } catch (e) {
+      const error = e instanceof Error ? e : new Error('Mailgun failed to validate emails')
+      sendToSentry(error, {tags: {type: 'Mailgun error'}})
+      return false
```

Comment:
-1 Why should this method have the return type `Promise<false | MailgunValidationResult>`, we could just return `unknown` here.